packages/server/utils/analytics/analytics.ts

```diff
@@ -94,38 +89,31 @@ class Analytics {
     template: MeetingTemplate
   ) => {
     meetingMembers.forEach((meetingMember) =>
-      this.meetingEnd(
-        meetingMember.userId,
-        completedMeeting,
-        meetingMembers,
-        createMeetingTemplateAnalyticsParams(template)
-      )
+      this.meetingEnd(meetingMember.userId, completedMeeting, meetingMembers, template)
     )
   }
 
   private meetingEnd = (
     userId: string,
     completedMeeting: Meeting,
     meetingMembers: MeetingMember[],
+    template?: MeetingTemplate,
     meetingSpecificProperties?: any
   ) => {
-    const {facilitatorUserId, meetingNumber, meetingType, phases, teamId} = completedMeeting
-    const presentMemberUserIds = meetingMembers.map(({userId}) => userId)
-    const wasFacilitator = userId === facilitatorUserId
     this.track(userId, 'Meeting Completed', {
-      hasIcebreaker: phases[0]?.phaseType === 'checkin',
-      // include wasFacilitator as a flag to handle 1 per meeting
-      wasFacilitator,
```

Comment:
+1 are we intentionally losing `wasFacilitator` here?

Reply:
That's right. We already have the `facilitatorUserId` so we can just use that

Reply:
That makes sense for data warehouse queries since we can just do `facilitatorUserId = userId`, but is there a good way to do this in other tools that Segment will send this to?  E.g. slicing Amplitude data on whether the user was the facilitator